src/interpret.rs

```diff
@@ -129,6 +130,17 @@ fn configure_working_directory(
     )
 }
 
+fn configure_volumes(
+    cmd: DockerCommandBuilder,
```

Comment:
I think because Rust defaults to move (instead of clone), nothing is cloned (and the move prevents the "out-of-date" value being used after an update). A `mut ref` might be preferable.

I also don't really like this API, if I'm honest. I've had in the back of my mind for a while to rework it.

Reply:
Rust's default use of move means there are no copies here, but I also don't really like the API, so I raised #37 to cover this.